ci/goinstall.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-VERSION="1.17.9"
+VERSION="1.17.10"
```

Comment:
you also need to update the build workflow files to use the correct version. For example [here](https://github.com/hashicorp/boundary/blob/3991a9fdb1d6c298383c0e81fb228b6c7c81693c/.github/workflows/build.yml#L23) and there are others in that file, just search for `go-version` or do a search and replace for `1.17.9` to `1.17.10`.

Reply:
Sorry about that, it's in my fixup commit- missed this file the first time